rclcpp/src/rclcpp/executors/static_single_threaded_executor.cpp

```diff
@@ -58,15 +81,21 @@ StaticSingleThreadedExecutor::add_node(
   if (has_executor.exchange(true)) {
     throw std::runtime_error("Node has already been added to an executor.");
   }
-
   if (notify) {
     // Interrupt waiting to handle new node
     if (rcl_trigger_guard_condition(&interrupt_guard_condition_) != RCL_RET_OK) {
       throw std::runtime_error(rcl_get_error_string().str);
     }
   }
-
```

Comment:
These changes to remove whitespace seem unnecessary, I'd recommend avoiding changes like this.